### DISCUSSION TITLE
Consider bringing back `v` prefix to tags/releases

### DISCUSSION THREAD

**Discussion by mmlb** [helix-editor/helix#4968] [General] [answered]
Hi, 

So I'm running helix via the nix flake. While doing so I noticed that nix thinks hx is at version v0.6.0 still even though there have been multiple releases since. I'm pretty sure this is because v0.6.0 was the last release with the `v` prefix as the other tags are regular tags and not light weight tags (that GH releases do when GH creates the tags). I know there's some ambiguity re semver and a `v` prefix in tags, but I'll say that I think the vast majority of projects I run into tag witht eh `v` prefix always.

**Comment by archseer** [comment, marked as answer]
It's showing `0.6.0` because that's what the `Cargo.toml` versions say. https://github.com/helix-editor/helix/blob/e92651816dea9f28ff2d73b06dedbe03c02ebe30/helix-term/Cargo.toml#L3

**Reply by mmlb** [reply]
Ah! I should have done better searching instead of going with my first hit. Is the outdated version in Cargo.toml a bug then?

**Reply by the-mikedavis** [reply]
I don't really see an advantage to updating the crate versions unless they're being consumed by outside packages/projects. Cargo enforces semantic versioning and we're using calendar versioning which conflict a bit

**Reply by mmlb** [reply]
Thats fair, and not really a big deal especially if helix's version scheme conflicts with cargo/semver semantics.
